scripts/generateBluegemJson.ts

```diff
@@ -0,0 +1,103 @@
+/// <reference lib="deno.ns" />
```

Comment:
Could we move this to `tools/generate_bluegem_json.ts`? `scripts` is somewhat of an overloaded term in a JS/TS repo.